backend/plugins/sales_api/src/modules/sales/db/models/Checklists.ts

```
@@ -0,0 +1,245 @@
+        { checklistId: currentItem.checklistId, order: destinationOrder },
+        { $set: { order: currentItem.order } },
+      );
+
+      await models.ChecklistItems.updateOne(
+        { _id },
+        { $set: { order: destinationOrder } },
+      );
+
+      return models.ChecklistItems.findOne({ _id }).lean();
+    }
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)